nexus-repository-conan/src/main/java/org/sonatype/repository/conan/internal/ui/ConanBrowseNodeGenerator.java

```diff
@@ -53,14 +53,14 @@ public ConanBrowseNodeGenerator() {
   private List<BrowsePaths> assetSegment(final String path) {
     String[] split = path.split("/");
     if(path.contains("packages")) {
-      if (split.length == 6) {
+      if (split.length == 8) {
         return BrowsePaths
-            .fromPaths(ImmutableList.of(split[split.length - 2], split[split.length - 1]),
+            .fromPaths(ImmutableList.of(split[split.length - 3], split[split.length - 2], split[split.length - 1]),
                 false);
```

Comment:
use constants for indexes of segments

Reply:
Using constant only here also looks not good. Is it better now fe80b4f141fa1eab522dd7dfa64f9bb18862b303